server/routes/induction/create/additionalTrainingCreateController.ts

```diff
@@ -51,7 +50,7 @@ export default class AdditionalTrainingCreateController extends AdditionalTraini
       return res.redirect(`/prisoners/${prisonNumber}/create-induction/check-your-answers`)
     }
 
-    if (updatedInduction.workOnRelease.hopingToWork === YesNoValue.YES) {
+    if (updatedInduction.workOnRelease.hopingToWork === HopingToGetWorkValue.YES) {
```

Comment:
We were using the wrong enum before, though it worked because at runtime (javascript) it was doing string equality on `YES`